swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
-class Item:
-    pass
\ No newline at end of file
+import uuid
+
+class Item():
+
+    def __init__(self, id=None, condition=0):
+        if not id:
+            self.id = int(uuid.uuid4())
+        else:
+            self.id = id
+        self.condition = condition
+    
+    def get_category(self):
+        return "Item"
+
+
+    def __str__(self) -> str:
+        return f"An object of type Item with id {self.id}."
```

Comment:
So, could we just call {self.get_category} inside of the f-string?

Reply:
We're confused because get_category is in the Item class, but not in Vendor.py so if we made a super in the Item class, would that then affect the Decor.py, electronics.py, etc?